src/view/com/util/List.web.tsx

```diff
@@ -72,23 +73,33 @@ function ListImpl<ItemT>(
     )
   }
 
-  let header: JSX.Element | null = null
+  let headerComponent: JSX.Element | null = null
   if (ListHeaderComponent != null) {
     if (isValidElement(ListHeaderComponent)) {
-      header = ListHeaderComponent
+      headerComponent = ListHeaderComponent
     } else {
       // @ts-ignore Nah it's fine.
-      header = <ListHeaderComponent />
+      headerComponent = <ListHeaderComponent />
     }
   }
 
-  let footer: JSX.Element | null = null
+  let footerComponent: JSX.Element | null = null
   if (ListFooterComponent != null) {
     if (isValidElement(ListFooterComponent)) {
-      footer = ListFooterComponent
+      footerComponent = ListFooterComponent
     } else {
       // @ts-ignore Nah it's fine.
-      footer = <ListFooterComponent />
+      footerComponent = <ListFooterComponent />
+    }
+  }
+
+  let emptyComponent: JSX.Element | null = null
```

Comment:
I opted to call this `emptyComponent` even though our other elements are just `header` for `ListHeaderComponent` and `footer` for `ListFooterComponent`, since just `empty` is a little confusing at first glance. I renamed the other two accordingly.